src/analysis/return_value.rs

```diff
@@ -63,6 +78,7 @@ pub fn analyze(env: &Env, func: &library::Function, type_tid: library::TypeId,
         parameter: parameter,
         base_tid: base_tid,
         commented: commented,
+        bool_return_is_error: if bool_return_is_error { Some(func.c_identifier.as_ref().unwrap().clone()) } else { None },
```

Comment:
g_setenv describes the function that actually failed (it's not the Rust code that failed), and glib::setenv might not actually exist (it might be exported as glib::some_module::setenv instead for example).

Allowing the config value as a string could make sense (it would allow to provide a more useful error string per function), and maybe allow a boolean as fallback which then does the above?